apps/blog/src/styles/blog.css

```
@@ -0,0 +1,615 @@
+@font-face {
+	font-family: "PlusJakartaSans";
```

Stylelint (17.4.0)

[error] Unexpected quotes around "PlusJakartaSans" (font-family-name-quotes)

(font-family-name-quotes)